platform/logging/LogLevel.ts

```diff
@@ -0,0 +1,55 @@
+import { assertNever } from '../lib/utils';
```

Comment:
`logging` folder currently co-exists with `logger` folder, I keep them both for now.